### DISCUSSION TITLE
Community Weekly Meeting (December 29, 2021)

### DISCUSSION THREAD

**Discussion by juzhiyuan** [apache/apisix#5954] [General]
Hi, here is the schedule for the coming Weekly Meeting.

**Schedule**

- **Date**: on Wednesday, December 29, at 20:00 ~ 21:30 (UTC +8)
- **Meeting Link**: https://meeting.tencent.com/dm/VLjTH3CU3QVi
- **Topics**: Please leave your comment below :)

**Note**
- If you have any questions, please comment below or contact **juzhiyuan@apache.org**
- Kindly let me know if you need **Google Meet** or **Zoom** as communication tools.

**Previous Meeting**
- https://github.com/apache/apisix/discussions/5873

**Comment by hanf2x** [comment]
Please upload to youtube also, Thanks.

**Reply by juzhiyuan** [reply]
sure! will let you know!

**Comment by juzhiyuan** [comment]
# Meeting Summary

- Video: https://www.bilibili.com/video/BV1yZ4y1D7gc/
- Docs: [Chinese](https://docs.qq.com/doc/DTkd3a2RraXBCaklm)
